lib/address.ex

```diff
@@ -14,7 +14,7 @@ defmodule Bitcoinex.Address do
     * p2wpkh: Pay-to-Witness-Public-Key-Hash
     * p2wsh: Pay-To-Witness-Script-Hash
   """
-  @type address_type :: :p2pkh | :p2sh | :p2wpkh | :p2wsh
+  @type address_type :: :p2pkh | :p2sh | :p2wpkh | :p2wsh | :p2tr
```

Comment:
We can consider using 

` @type address_type :: unquote(AltoSupport.ElixirUtils.Macro.list_to_union_type_ast(@address_types))`

so we never need to manually update the typespec to in sync with module attributes